decidim-meetings/app/commands/decidim/meetings/admin/update_registrations.rb

```diff
@@ -0,0 +1,44 @@
+# frozen_string_literal: true
+
+module Decidim
+  module Meetings
+    module Admin
+      # This command is executed when the user updates the meeting registrations.
+      class UpdateRegistrations < Rectify::Command
+        # Initializes a UpdateRegistrations Command.
+        #
+        # form - The form from which to get the data.
+        # meeting - The current instance of the meeting to be updated.
+        def initialize(form, meeting)
+          @form = form
+          @meeting = meeting
+        end
+
+        # Updates the meeting if valid.
+        #
+        # Broadcasts :ok if successful, :invalid otherwise.
+        def call
+          @meeting.with_lock do
+            return broadcast(:invalid) if @form.invalid?
```

Comment:
Do we need the lock to check if the form is valid? If we don't, swap lines 21 and 22.